bootstrap/cmd/kfdefConverter/README.md

```diff
@@ -0,0 +1,23 @@
+# KfDef version converter
+
+## Overview
+
+This is a simple helper CLI that converts KfDef between versions.
+
+## Usage
+
+```
+A simple CLI to convert KfDef from v1alpha1 to v1beta1
+
+Usage:
+  kfdef-converter [command]
+
+Available Commands:
+  help        Help about any command
+  tov1beta1   Convert a KfDef config in v1alpha1 into v1beta1 format.
+
```

Comment:
probably add an example usage of kfdef-converter tov1beta1

Reply:
Added.